Tools/Scripts/webkitpy/benchmark_runner/benchmark_builder.py

```diff
@@ -95,6 +103,32 @@ def _checkout_with_subversion(self, subversion_url):
         if error_code:
             raise Exception('Cannot checkout the benchmark - Error: %s' % error_code)
 
+    def _download_from_github(self, github_source, github_subtree):
+        _log.info(f'Downloading content from {github_source}')
+        GithubDownloadTask(github_source, github_subtree).execute(self._dest)
+
+    def _local_git_archive_eligible(self):
+        if 'local_git_archive' not in self._plan:
+            return False
+        if not self.LOCAL_GIT_ARCHIVE_SCHEMA.match(self._plan['local_git_archive']):
+            return False
+        is_git_checkout = subprocess.check_output(['git', '-C', os.path.dirname(__file__), 'rev-parse',
```

Comment:
will this fail if the path doesn't exist?

Reply:
No, because the path doesn't exist in current commit does not necessarily mean it doesn't exist in specified commit.
Since the plan file is manually created by us, if anything goes wrong, we should be able to notice and fix the issue very soon.

Reply:
Per discussion `os.path.dirname(__file__)` will always work unless we archive entire python into a zip executable.